corehq/blobs/tests/test_migratingdb.py

```diff
@@ -20,16 +18,11 @@ class TestMigratingBlobDB(get_base_class()):
 
     @classmethod
     def setUpClass(cls):
-        with trap_extra_setup(AttributeError, msg="S3_BLOB_DB_SETTINGS not configured"):
-            config = settings.S3_BLOB_DB_SETTINGS
-        cls.s3db = TemporaryS3BlobDB(config)
-        cls.fsdb = TemporaryFilesystemBlobDB()
-        cls.db = mod.MigratingBlobDB(cls.s3db, cls.fsdb)
+        super(TestMigratingBlobDB, cls).setUpClass()
 
-    @classmethod
-    def tearDownClass(cls):
-        cls.fsdb.close()
-        cls.s3db.close()
+        cls.s3db = cls.db
```

Comment:
nitpick: maybe add `assert isinstance(cls.db, TemporaryS3BlobDB), cls.db` here to protect against the base class changing in some unanticipated way?